packages/evm-connector/signature-verification/Cargo.lock

```diff
@@ -0,0 +1,4511 @@
+# This file is automatically @generated by Cargo.
```

Comment:
As we removed all the `.rs` files we can remove also the Cargo.lock and Cargo.toml and other unused files.